tool/teleport/common/teleport.go

```diff
@@ -489,13 +490,14 @@ func normalizeOutput(output string) string {
 }
 
 func checkConfigurationFileVersion(version string) error {
```

Comment:
What's the difference between this and the `validateVersion` func in `validateconfig.go`? Do we need both?